work/flow.rst

```diff
@@ -323,6 +323,15 @@ Review Preparation
 When development on your ticket branch is complete, we use a standard process for reviewing and merging your work.
 This section describes how to prepare your work for review.
 
+.. _workflow-search-todo:
+
+Search for TODO comments
+------------------------
+
+Search for your ticket number in all repositories associated with the LSST organization on Github (using "org:lsst" as the prefix).
```

Comment:
This seems like a reasonably quick and useful step.  Are you confident that there will not be many false positive hits?  Are you confident that this is uncontroversial enough that we don't need an RFC for it?  How are you planning on announcing this?

Reply:
I occasionally do search for `DM-NNNNN` and I rarely see any false positives (just searching for the number does give a lot of false positives though). May be I should clarify to include the `DM-` prefix. 

I think an announcement on the #dm channel would suffice. I wasn't thinking that this would require RFC, but can do one if you think it'll get it noticed more.

I do think it's non-controversial since it's a quick step and it is applicable only when a ticket has been lingering around for a long time. That's something I should add as well.